test/EFCore.PG.FunctionalTests/EFCore.PG.FunctionalTests.csproj

```diff
@@ -19,7 +18,9 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\EFCore.PG\EFCore.PG.csproj" />
+<!--
     <ProjectReference Include="..\..\src\EFCore.PG.NTS\EFCore.PG.NTS.csproj" />
+-->
```

Comment:
Yeah, the spatial tests are commented out, and since the EFCore.PG.NTS project doesn't build at the moment, I removed this dependency so we can run all other tests.

Spatial support requires another layer of work, to port to the new NetTopologySuite (see #835)... This also likely involves changes at the ADO layer so I'm punting this to preview8.